Sources/Sentry/SentryScopeSyncC.c

```diff
@@ -0,0 +1,225 @@
+#include <pthread.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define NUMBER_OF_FIELDS 9
+
+static const char *userJSON;
+static const char *distJSON;
+static const char *contextJSON;
+static const char *environmentJSON;
+static const char *tagsJSON;
+static const char *extrasJSON;
+static const char *fingerprintJSON;
+static const char *levelJSON;
```

Comment:
Make sure to initialize these to `= NULL`, otherwise they are undefined, and can segfault when you do a `getSize` down below.